schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -171,6 +172,7 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2014-08-01-preview/Microsoft.Scheduler.json#/resourceDefinitions/jobCollections" },
                   { "$ref": "https://schema.management.azure.com/schemas/2015-03-01-preview/Microsoft.AppService.json#/resourceDefinitions/gateways" },
                   { "$ref": "https://schema.management.azure.com/schemas/2015-04-01/Microsoft.NotificationHubs.json#/resourceDefinitions/namespaces" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2015-04-01/Microsoft.NotificationHubs.json#/resourceDefinitions/namespaces_authorizationRules" },
```

Comment:
>namespaces_authorizationRules [](start = 137, length = 29)

can you please update 2014-04-01-preview/deploymentTemplate.json as well?

Reply:
Done